lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -3055,8 +3055,8 @@ void CommandInterpreter::PrintCommandOutput(IOHandler &io_handler,
   }
 
   std::lock_guard<std::recursive_mutex> guard(io_handler.GetOutputMutex());
-  if (had_output && INTERRUPT_REQUESTED(GetDebugger(), 
-                                        "Interrupted dumping command output"))
+  if (had_output &&
+      INTERRUPT_REQUESTED(GetDebugger(), "Interrupted dumping command output"))
```

Comment:
revert non related whitespace change

Reply:
No, it is actually required by the formatter reported by github: 
https://github.com/llvm/llvm-project/actions/runs/7789051206/job/21239805569?pr=80375